src/index.html

```diff
@@ -8,16 +8,42 @@
     <link rel="stylesheet" href="styles/styles.css" />
   </head>
   <body>
-    <header class="app-header">
-      <h1 class="app-header__heading">Quiz App</h1>
+    <header class="header">
+      <div class="flex mx-4 my-1 flex-align-center">
```

Comment:
Can you not give these styles the `header` element directly?

Reply:
true 🤦🏼